source/includes/harvest/_demographic_data.md

```diff
@@ -0,0 +1,463 @@
+    }
+  ]
+}
+```
+
+### Noteworthy attributes
+
+| Attribute | Description |
+|-----------|-------------|
+| id | The demographic question's unique identifier |
+| translations.language | Only `en` (English) is supported at this time.
+| translations.name | The question text
+| active | If `false`, the question has been deleted.
+
+## The Demographic Answer Option object
+
+```json
+{
+  "id": 456,
+  "free_form": false,
+  "active": true,
+  "demographic_question_id": 123,
+  "translations": [
+    {
+      "language": "en",
+      "name": "Blue"
+    }
+  ]
+}
+```
+
+### Noteworthy attributes
+
+| Attribute | Description |
+|-----------|-------------|
+| id | The demographic answer option's unique identifier |
+| translations.language | Only `en` (English) is supported at this time.
+| translations.name | The answer option text.
+| active | If `false`, the answer option has been deleted.
+| free_form | If `true`, the answer option allows free-form user input.
+| demographic_question_id | The demographic question for which the answer option belongs to.
+
+## The Demographic Answer object
+
+```json
+{
+  "id": 51,
+  "free_form_text": "12am",
+  "application_id": 107594341,
+  "demographic_question_id": 25,
+  "demographic_answer_option_id": 106,
+  "created_at": "2019-04-29T18:46:03.707Z",
+  "updated_at": "2019-04-29T18:46:03.707Z"
+}
+```
+
+### Noteworthy attributes
+
+| Attribute | Description |
+|-----------|-------------|
+| id | The demographic answer's unique identifier |
+| free_form_text | If the selected answer option is free-form, this is the value given by the user.  Otherwise `null`.
+| application_id | The application for which the demographic question was answered
+| demographic_question_id | The demographic question which was answered.
+| demographic_answer_option_id | The demographic answer option which was selected.
+
+## GET: List Demographic Questions
+
+List all of an organization's demographic questions.
+
+```shell
+curl 'https://harvest.greenhouse.io/v1/demographics/questions'
+-H "Authorization: Basic MGQwMzFkODIyN2VhZmE2MWRjMzc1YTZjMmUwNjdlMjQ6"
+```
+
+> The above command returns JSON structured like this:
+
+```json
+[
+  {
+    "id": 123,
+    "translations": [
+      {
+        "language": "en",
+        "name": "What is your favorite color?"
+      }
+    ]
+  },
+  {
+    "id": 456,
+    "translations": [
+      {
+        "language": "en",
+        "name": "Pizza or pasta?"
+      }
+    ]
+  }
+]
+```
+
+### HTTP Request
+
+`GET https://harvest.greenhouse.io/v1/demographics/questions`
+
+### Querystring parameters
+
+| Parameter | Description |
+|-----------|-------------|
+*per_page | Return up to this number of objects per response. Must be an integer between 1 and 500. Defaults to 100.
+
+<br>
+[See noteworthy response attributes.](#the-demographic-question-object)
+
+This endpoint supports pagination. See the [Pagination](#pagination) section for more detail.
+
+## GET: Retrieve Demographic Question
```

Comment:
Maybe this is just a Github issue, but the formatting from here down seems a little weird. Is there a missing ` or something?

Reply:
Yeah it looks like the Github markdown syntax highlighter expects table rows the start with `|`, but they are optional for our purposes.  Fixed!